rcd/rustdesk-hbbs

```diff
@@ -9,10 +9,10 @@
 #
 # rustdesk_hbbs_enable (bool):            Set to NO by default.
 #               Set it to YES to enable rustdesk_hbbs.
-# rustdesk_hbbs_ip (string):              Set IP address/hostname of relay server to use
+# rustdesk_hbbr_server (string):          Set IP address/hostname of relay (hbbr) server to use
```

Comment:
This is actually wrong. All the variables in this script must start with `rustdesk_hbbs`, since this is the name of the script. This is documented in rc.subr(8) and [here](https://docs.freebsd.org/en/articles/rc-scripting/)

Reply:
BTW, having variables starting with the same name of another script that does exist in rc.d would be very confusing for most users. Much more confusing than `rustdesk_hbbs_ip` is. At least with `rustdesk_hbbs_ip` it would be simple to know in which script to look for explanations. by naming the variable `rustdesk_hbbr_server` you end up misdirecting people to look in the wrong file.

Reply:
I see your point more clearly now. I will close this PR.